test/inductor/test_halide.py

```diff
@@ -0,0 +1,86 @@
+# Owner(s): ["oncall: pt2"]
+import textwrap
+import unittest
+
+import torch
+from torch._inductor.codecache import HalideCodeCache
+from torch._inductor.runtime.hints import HalideInputSpec, HalideMeta
+from torch._inductor.test_case import run_tests, TestCase
+from torch._inductor.utils import parallel_num_threads
+
+from torch.testing._internal.common_utils import IS_MACOS
+from torch.testing._internal.inductor_utils import HAS_CPU
+
+
+try:
+    import halide
+
+    HAS_HALIDE = halide is not None
+except ImportError:
+    HAS_HALIDE = False
+
+
+@unittest.skipUnless(HAS_HALIDE, "requires halide")
+class HalideTests(TestCase):
+    def test_codecache(self):
+        fn = HalideCodeCache.generate_halide(
+            HalideMeta(
+                argtypes=[
+                    HalideInputSpec(ctype="float*", name="in_ptr0", numel="1024L"),
+                    HalideInputSpec(ctype="float*", name="in_ptr1", numel="1024L"),
+                    HalideInputSpec(
+                        ctype="float*",
+                        name="out_ptr0",
+                        numel="1024L",
+                    ),
+                ],
+                target="host",
+                scheduler="Mullapudi2016",
```

Comment:
just curious - why this scheduler the others ?

Reply:
It is a placeholder while I focus on functionality.